.ci_scripts/addX509User.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+
+set -e -x
+
+echo
+echo "adding x509 Certificate subject as a User"
+# https://docs.mongodb.com/manual/tutorial/configure-x509-client-authentication/#add-x-509-certificate-subject-as-a-user
+
+PATH="$1"
+DB_PATH="$2"
+SCRIPT_DIR="$3"
+ENV_FILE="$4"
+
+PORT=27018
```

Comment:
Would rather get it from `PRIMARY_HOST` available in `setupEnv.sh`

Reply:
:+1: